app/Orchid/Screens/Accomodation/AccomodationEditScreen.php

```diff
@@ -128,9 +128,14 @@ public function layout(): iterable
                         ->title('no'),
                 ]),
 
+                Input::make('accomodation.accommodation_number')
```

Comment:
Can this take multiple images? or just one?

Reply:
multiple images can be uploaded for `Accomodation`